src/components/admin/services/timer/controls.tsx

```diff
@@ -2,26 +2,20 @@ import { Trash2, Pen, Check } from "lucide-react";
 
 type props = {
   edit: boolean;
-  setEdit: (value: boolean) => void;
   onRemove: () => void;
 };
 
-const Controls = ({ edit, setEdit, onRemove }: props) => {
+const Controls = ({ edit, onRemove }: props) => {
   return (
     <div className="flex gap-4">
```

Comment:
what's the point of having the edit? it seems like no matter the condition it will render the same thing? so we dont need to pass in the edit variable here